sdk/formrecognizer/Azure.AI.FormRecognizer/src/SubscriptionProperties.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.AI.FormRecognizer.Custom
+{
+    public class SubscriptionProperties
+    {
+        internal SubscriptionProperties(ModelsSummary_internal summary)
```

Comment:
Is there a difference between these two types?

Reply:
Property name LastUpdatedDateTime -> LastUpdatedOn